lib/elastic_apm/config.rb

```diff
@@ -98,21 +105,21 @@ def initialize(options = nil)
     def app=(app)
```

Comment:
I think `app` is closer to the ruby way of things. I want it to be easiest for devs and I think more people will be confused by `service` than by `app`?

Reply:
I thought we are trying to introduce `service` as a general name in the elastic apm world instead of `app`, and as the config options also are named `service` it might be confusing.

Reply:
Leaving as is for now. Considering a rename 👍